zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/utils/SearchDatabaseCallbackDelegate.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.exporter.utils;
+
+import io.camunda.exporter.config.ConnectionTypes;
+import io.camunda.exporter.config.ExporterConfiguration;
+import java.util.Map;
+import org.junit.jupiter.api.extension.AfterAllCallback;
+import org.junit.jupiter.api.extension.AfterEachCallback;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+
+public interface SearchDatabaseCallbackDelegate
```

Comment:
:wrench: Similarly, if we are going with this approach, we should understand how this is meant to be implemented and used in the future. e.g. do I have to implement after all? before each? What should I implement for `contextAdapterRegistration`, and when is it called?

So I would strongly urge to document this interface (as we should in general document interface/abstract types for similar reasons)

Reply:
Yes, absolutely agree. I will do that before the merge.